reference/glossary.md

```diff
@@ -0,0 +1,445 @@
+
+<a name="appium"></a>
+### Appium
+
+An open-source test automation tool that can be used to test native, mobile and hybrid web applications on Android.
+
+See [the Appium website](http://appium.io/).
+
+<a name="application"></a>
+### Application
+
+One of the main objects of Anbox Cloud. An application encapsulates an Android app and manages it within the Anbox Cloud cluster.
+
+See https://discourse.ubuntu.com/t/about-applications/17760.
+
+<a name="application-container"></a>
+### Application container
+
+A container that is created when launching an application.
+
+See [Application containers vs. raw containers](https://discourse.ubuntu.com/t/about-containers/17763#application-vs-raw).
+
+<a name="application-manifest"></a>
+### Application manifest
+
+A file that defines the attributes of an Anbox Cloud application.
+
+See https://discourse.ubuntu.com/t/application-manifest/24197.
+
+<a name="base-container"></a>
+### Base container
+
+A temporary container that is used when bootstrapping an application. It is automatically deleted when the application bootstrap is completed.
+
+See https://discourse.ubuntu.com/t/about-containers/17763.
+
+<a name="boot-package"></a>
+### Boot package
+
+The package to launch in an application container once the system has booted.
+
+<a name="bootstrap-process"></a>
+### Bootstrap process
+
+The process that builds the application and synchronises it across all LXD nodes in the cluster when creating an application.
+
+See [Bootstrap process](https://discourse.ubuntu.com/t/about-applications/17760#bootstrap).
+
+<a name="container"></a>
+### Container
+
+One of the main objects of Anbox Cloud. Every time you launch an application or image, Anbox Cloud creates a container for it. Every container provides a full Android system.
+
+See https://discourse.ubuntu.com/t/about-containers/17763.
+
+<a name="core-stack"></a>
+### Core stack
+
+The core parts of the Anbox Cloud stack that are required for all deployments.
+
+See https://discourse.ubuntu.com/t/about-anbox-cloud/17802.
+
+<a name="coturn"></a>
+### Coturn
+
+An open-source implementation of a STUN/TURN server needed for WebRTC to work behind NATs and firewalls.
+
+See [the Coturn project on GitHub](https://github.com/coturn/coturn).
+
+<a name="grafana"></a>
+### Grafana
+
+A tool for analytics and monitoring that allows to query and visualise the metrics of the cluster or individual containers and generate alerts based on the data.
+
+See https://discourse.ubuntu.com/t/use-grafana/17787 and [the Grafana website](https://grafana.com/).
+
+<a name="gpu"></a>
+### Graphics Processing Unit (GPU)
+
+A specialised processor that is designed to accelerate image processing and graphics rendering for output to a display device.
+
+<a name="ha"></a>
+### High availability (HA)
+
+The characteristic of a system to continuously be available without failing for a higher-than-normal period of time.
```

Comment:
While that definition is correct, I would be a bit more specific to what it means in our case here. In our case HA means avoiding having a single point of failure by having replicas of every service